apps/custom-ui-floating-window-demo/pages/api/getContacts.js

```diff
@@ -0,0 +1,42 @@
+import { getCookie } from "cookies-next";
+import logger from "../../shared/logger";
+import { PersonsApi } from "pipedrive";
```

Comment:
try to order node_modules imports before local imports

```suggestion
import { PersonsApi } from "pipedrive";

import logger from "../../shared/logger";
```